thirdparty/ossp-uuid/ossp-uuid-update-config-guess.patch

```diff
@@ -0,0 +1,5538 @@
+    diff --git a/config.guess b/config.guess
+index ca2a03c..69188da 100755
+--- a/config.guess
++++ b/config.guess
```

Comment:
Where is this huge patch coming from?

Reply:
They are the newer version of the included config.guess and config.sub
https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess
https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub

Reply:
Can we update the whole package instead?